charts/tidb-operator/templates/scheduler-deployment.yaml

```diff
@@ -26,6 +26,10 @@ spec:
         app.kubernetes.io/name: {{ template "chart.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
         app.kubernetes.io/component: scheduler
+      annotations:
```

Comment:
How about putting this line into the `if` block?

Reply:
Seems better, will do that in a bit